src/__tests__/api/admin/permissions.test.ts

```
@@ -0,0 +1,181 @@
+  ],
+}));
+
+const mockSession = {
+  user: {
+    id: "admin-user-id",
+    email: "admin@example.com",
+    role: "ADMIN",
+  },
+};
+
+const mockAbility = {
```

Biome (2.1.2)

[error] Shouldn't redeclare 'mockAbility'. Consider to delete it or rename it.

'mockAbility' is defined here:


(lint/suspicious/noRedeclare)